### DISCUSSION TITLE
Adding a new model.

### DISCUSSION THREAD

**Discussion by boykan313** [Purfview/whisper-standalone-win#26] [Q&A] [answered]
Greetings! Help add a new model, please.
I want to add this one: https://huggingface.co/mitchelldehaven/whisper-large-v2-ru
since this model is for a regular whisper, we need to convert e for whisper-faster. In my opinion, I succeeded with the help of this command, but I don’t understand how to use it further.

ct2-transformers-converter --model mitchelldehaven/whisper-large-v2-ru --output_dir whisper-large-v2-ru-ct2 --copy_files tokenizer.json --quantization float16

I was advised to convert it, I seem to have succeeded with the help of these commands, but I don’t know how to add it. Support information please!
![322221](https://github.com/Purfview/whisper-standalone-win/assets/90963133/32c92313-79a0-452c-af1c-365495a86767)

**Comment by Purfview** [comment, marked as answer]
Have model in `_models\faster-whisper-large-v2-ru` folder and use `--model=large-v2-ru`

**Comment by boykan313** [comment]
> faster-whisper-large-v2-ru

![3212](https://github.com/Purfview/whisper-standalone-win/assets/90963133/44349bcc-f27b-4e50-b011-034e6879bf5e)
in that case i get this error.

**Comment by boykan313** [comment]
the model that I want to put is for the usual whisper, I managed to convert it with the command that I indicated above, but I don’t understand how to proceed. I was advised to use this command, but I do not understand how to use it.
"Once the model is converted, you can load it from its path on disk:

model = WhisperModel("/path/to/converted/model") "

**Comment by Purfview** [comment]
Show what you get with `dir /S` command.

**Reply by boykan313** [reply]
![1](https://github.com/Purfview/whisper-standalone-win/assets/90963133/e0c6e857-0e61-46a0-a9c6-c97deee66817)
![2](https://github.com/Purfview/whisper-standalone-win/assets/90963133/b48f5f25-21cf-4797-bd73-5af2bc01d29f)
![3](https://github.com/Purfview/whisper-standalone-win/assets/90963133/834b3e99-c3fc-4ba0-b43d-9e66ff914110)

**Reply by boykan313** [reply]
in which folder should I use this command?

**Comment by Purfview** [comment]
Model there is named `pytorch_model.bin`, it should be named `model.bin`

**Reply by boykan313** [reply]
![321111](https://github.com/Purfview/whisper-standalone-win/assets/90963133/2c95e5a4-1f72-43b9-a688-7bd538164913)
Did that, now I'm getting this kind of error....

**Reply by Purfview** [reply]
Dunno what this error means, post this screenshot there: -> https://github.com/guillaumekln/faster-whisper/issues/314

EDIT: I did that for you.

**Comment by boykan313** [comment]
> `dir /S`



> Show what you get with `dir /S` command.

Here is where the problem lies. I was advised in the whisper-faster thread to do this. I did, but I can’t do the download, I don’t understand where to insert the second code.

См. раздел «Преобразование модели» в README, чтобы преобразовать модель для более быстрого шепота: [https://github.com/guillaumekln/faster-whisper#model-conversion .](https://github.com/guillaumekln/faster-whisper#model-conversion)

После преобразования модели вы можете загрузить ее из пути на диске:

model = WhisperModel("/path/to/converted/model")

**Reply by Purfview** [reply]
Don't ask me about issues you have with other repos. Ask there.
